### PR TITLE
Crudl security update

### PR DESCRIPTION
**Pull request by stafyniaksacha**
Add replace / update actions on security document
The inline view now update instead of replace elements

### CODE REVIEW

File: public/javascripts/common/authentication/loginModal.controller.js [hunk 1]
```diff
@@ -5,11 +5,13 @@ angular.module('kuzzle.authentication')
   '$log',
   'AUTH_EVENTS',
   'AuthService',
-function ($scope, $uibModal, $log, AUTH_EVENTS, Auth) {
+  '$state',
+function ($scope, $uibModal, $log, AUTH_EVENTS, Auth, $state) {
   var modalInstance = null;
 
   var showDialog = function (nextState) {
```
Comment (xbill82) [inline public/javascripts/common/authentication/loginModal.controller.js]: We no longer need `nextState` here.

**Comment by dbengsch** [comment]
+1

File: public/javascripts/common/documentsInline/userToolbar.directive.js [hunk 2]
```diff
@@ -21,14 +21,19 @@ angular.module('kuzzle.documentsInline')
...
       $scope.saveEditDocument = function () {
-        var user = {};
+        var user = {
+          id: $scope.document.id,
+          content: {}
+        };
 
         try {
-          $scope.document.content = JSON.parse($scope.document.json);
+          user.content = JSON.parse($scope.document.json);
```
Comment (xbill82) [inline public/javascripts/common/documentsInline/userToolbar.directive.js]: What if `$scope.document.json` is `undefined` here?

**Comment by xbill82** [comment]
:+1:

**Pull request merged**